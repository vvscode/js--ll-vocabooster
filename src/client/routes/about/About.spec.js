import React from 'react';
import { shallow } from 'enzyme';
import Component from './About.component';

describe('About', () => {
  let props;
  let getShallowComponent;

  beforeEach(() => {
    props = {
      actions: {},
      route: {},
    };

    getShallowComponent = () => shallow(<Component {...props} />);
  });

  describe('render', () => {
    it('should render with all props', () => {
      expect(getShallowComponent()).toMatchSnapshot();
    });
  });
});
/*
File was generated with tamm-cli from next settings:
{
  "path": "about",
  "addStyles": true,
  "connectedToRedux": "No",
  "componentName": "about",
  "camelizedComponentName": "About",
  "withRedux": false
}
*/
