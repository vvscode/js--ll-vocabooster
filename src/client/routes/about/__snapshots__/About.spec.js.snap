// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About render should render with all props 1`] = `
<div
  className="about__container"
>
  <div
    className="about__wrapper"
  >
    <_class
      footer={
        <h3>
          About footer
        </h3>
      }
      header={
        <h2>
          About header: 
        </h2>
      }
      route={Object {}}
    />
  </div>
</div>
`;
